HW03.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "HW03"
+author: "Laura Hunter"
+date: "7/19/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library("ggplot2")
+library("gridExtra")
+```
+
+## Timing Score Plots
+
+In the [neuroeconomics lab](https://www.gottlieblab.com/) where I worked as a research technician, I created a project based on [this paper](https://9fabbb78-5b5b-4a1b-aeb0-b5443b6eac2f.filesusr.com/ugd/9cf124_28146e8d3fa5431baaa796c288f0724c.pdf). It involves two lotteries, wherein the participant needs to make a choice between one that has a high level of variability in reward size (e.g. you win 100 points or 200 points) and one that has a low level of variability in reward size (e.g. you win 140 points or 160 points).
+
+The details of the experimental design are not important for understanding the timing scores, however. Once our data collection (ongoing) is complete, we expect about 200 participants, so I am scoring each participants' performance in order to remove those that clearly misunderstood the task.
+
+The participants had a minimum of 600 milliseconds and a maximum of 10 seconds to make a decision. In the dataset I'm loading here, I have calculated each participant's ability to do this task within that timing window (number of trials with bad timing over total number of trials). Once I did that, I Z-Scored them to get a feel for what we should expect from participants. This is in the hopes of knowing which participants were "too bad" at the task (too positive/far to the right) and therefore need to be removed from our datapool.
+
+I wanted to create a visualization of this, separated by block (there are 4 blocks). Here it is!
+
+```{r Timing Scores}
+
+#Read in our CSV of data (pre-cleaned... in MATLAB)
+data_scoring <- read.csv("2lot_data_scoring.csv", stringsAsFactors = FALSE)
+
+#Pre-allocate color choices for aesthetic flexibility
+sd_color <- "blue" #color of standard deviation line
+mean_color <- "red" #color of mean line
+fill_color <- "green"
+mean_linetype <- "dashed" #linetype of mean line
+sd_linetype <- "dashed" #linetype of standard deviation line
+
+#Put data into lists so that each plot can be made in a loop
+block_list <- list()
+data_list <- list(data_scoring$obs_bad_timing, data_scoring$rep_bad_timing, data_scoring$int_bad_timing, data_scoring$gue_bad_timing)
+title_list <- list("Observe Block", "Repeat Observe Block", "Intervene Block", "Guess Block")
+
```

Comment:
Very clever use of a foor loop. I probably would have done the plots one by one (and ended up with an unnecessarily long script)